tests/testsuite/cargo_owner/help/stdout.log

```diff
@@ -1,18 +1,19 @@
 Manage the owners of a crate on the registry
 
-Usage: cargo[EXE] owner [OPTIONS] [CRATE]
+Usage: cargo owner add    <OWNER_NAME> [CRATE_NAME] [OPTIONS]
```

Comment:
Should we add `add_help`, `remove_help`, and `list_help` UI tests?

Reply:
I initially thought these tests might be simple and redundant, so I didn't add them.